spec/controllers/scans_controller_spec.rb

```diff
@@ -115,13 +115,27 @@
       end
     end
     describe 'invalid requests' do
-      let(:user) { create(:webauth_user) }
+      let(:user) { create(:webauth_user).tap { |u| u.ldap_group_string = Settings.scan_pilot_groups.first } }
```

Comment:
I wonder if we should create a special FactoryGirl user factory for users who can scan and set this `ldap_group_string` there (so we can re-use that particular user logic around different tests), like the one in the `create_scan_spec`.